app/src/main/java/com/example/football_manager/News/ViewOneNews.kt

```diff
@@ -0,0 +1,109 @@
+package com.example.football_manager.News
```

Comment:
Looks good, can't test without completing the first comment.

Reply:
First comment done